ibm_catalog.json

```diff
@@ -729,21 +885,23 @@
             },
             {
               "key": "secrets_manager_service_plan",
-              "required": true,
+              "type_metadata": "string",
               "virtual": true,
+              "required": true,
               "type": "string",
+              "default_value": "standard",
               "options": [
                 {
+                  "description": "Centrally manage secrets in a dedicated instance.",
                   "displayname": "Standard",
                   "value": "standard"
                 },
                 {
+                  "description": "30-day free trial, only one trial instance provisioned in your account at any time.",
```

Comment:
why are we not using the same text as the service tile? I would include all of this text:

<img width="787" height="167" alt="Image" src="https://github.com/user-attachments/assets/e4a4e6be-2533-4da5-8469-ed12ea3efa30" />